modules/configuration-ovnk-network-plugin-json-object.adoc

```diff
@@ -37,7 +37,10 @@ For `"topology":"layer2"` deployments, IPv6 (`2001:DBB::/64`) and dual-stack (`1
 
 |`netAttachDefName`
 |`string`
-| The namespace and the network attachment name. The value must match the values specified for `namespace` and `name` when defining the network attachment definition object. For example, `ns1/l2-network` refers to the `ns1` namespace and the `l2-network` network attachment definition name.
+| The network attachment name. These networks are not namespaced. For example, you can have one `netAttachDefName` named `l2-network` on one `namespace` object, and a second `netAttachDefName` named `l2-network` on a second `namespace` object. This ensures that pods on different namespaces can communicate over a secondary overlay network.
+
```

Comment:
I find that the previous doc was fine as Network attachment definitions are namespaced.

I think this should go in the `name` row, not the `netAttachDefName` row.

Reply:
I agree w/ @jcaamano , good catch. We were making the docs worse. Good catch. I apologize for lgtm'ing this.

I think the clarification should indeed happen on the `name` attribute - this is the attribute that should **not** have anything to do with namespaces.